dspace-api/src/main/resources/org/dspace/storage/rdbms/sqlmigration/oracle/V7.6_2023.09.18__workspaceitem_add_item_id_unique_constraint.sql

```diff
@@ -0,0 +1,9 @@
+--
+-- The contents of this file are subject to the license and copyright
+-- detailed in the LICENSE and NOTICE files at the root of the source
+-- tree and available online at
+--
+-- http://www.dspace.org/license/
+--
+
+ALTER TABLE workspaceitem ADD CONSTRAINT unique_item_id UNIQUE(item_id);
```

Comment:
Please remove the Oracle migration as Oracle is no longer supported.

Reply:
I have removed the Oracle migration